scripts/linters/pylint_extensions.py

```diff
@@ -1959,6 +2026,386 @@ def process_tokens(self, tokens):
+    """
+
+    __implements__ = interfaces.ITokenChecker
+
+    ALLOWED_LINES_OF_GAP = 10
+
+    name = 'comment-for-exceptional-types'
+    priority = -1
+    msgs = {
+        'C0047': (
+            'Any type is used. Please add a proper comment if'
+            ' Any type is needed.',
+            'any-type-used',
+            'Annotations with Any type should be done with'
+            ' a proper comments.'
+        ),
+        'C0048': (
+            'cast function is used. Please add a proper comment if'
+            ' cast is needed.',
+            'cast-func-used',
+            'Casting of any value should be done with a proper comment.'
+        ),
+        'C0049': (
+            'object class is used. Please add a proper comment if'
+            ' object is needed.',
+            'object-class-used',
+            'Annotating any value with object should be done with a proper'
+            ' comment.'
+        )
+    }
+
+    def process_tokens(self, tokens):
+        """Custom pylint checker which makes sure that every exceptional type
+        should be documented properly.
+
+        Args:
+            tokens: Token. Object to access all tokens of a module.
+        """
+        self.check_comment_is_present_with_any_type(tokens)
+        self.check_comment_is_present_with_cast_method(tokens)
+        self.check_comment_is_present_with_object_class(tokens)
+
+    def _check_import_status(
+        self, import_status_vars, token_type, token, line_num
+    ):
+        """Checks whether the single-line import or multi-line import
+        present inside the module. If multi-line import is present then
+        it checks whether the linters are currently inside multi-line
+        import's scope or not.
+
+        Args:
+            import_status_vars: List[bool, int, bool]. This list contains
+                the variables that tracks the module's import status, where:
+                1st element of list: Indicates whether single line import
+                    is encountered or not.
+                2nd element of list: Indicates the line number if import
+                    is encountered, otherwise it is zero.
+                3rd element of list: Indicates whether the multi-line import is
+                    encountered and linters are in it's scope or not.
+                    import(
+                        << multi-line import's scope >>
+                    )
+            token_type: int. The kind to token that pylint provided.
+            token: str. The token of module the pylint provided.
+            line_num: int. The line number of given token.
+
+        Returns:
+            List[bool, int, bool]. The list which contains the information of
+            the current module's import status.
+        """
+        single_line_import = import_status_vars[0]
+        import_line_num = import_status_vars[1]
+        inside_multi_line_import_scope = import_status_vars[2]
```

Comment:
Can we use dict for string these values or namedtuple?

Reply:
> Can we use dict for string these values

Now we are using Dict -- PTAL!